### DISCUSSION TITLE
EXR export way overexposed, PNG fine

### DISCUSSION THREAD

**Discussion by sebarious70** [bezo97/IFSRenderer#44] [Q&A]
I don't know if this is a bug, since I just started getting to know this program. I came across it from watching this video: https://www.youtube.com/watch?v=Q51aTqcS63U

I tried to replicate as best as I could what the creator of the video was doing, and probably missing some things because my result was way different. But that's OK, because I came up with something that looks great anyway.

The problem seems to be the EXR export. It's completely overexposed, like bumping up the exposure to where there's a big blotch of white in the center of the graphic. I'm not familiar with the terms, so I'm going to just post examples below. Obviously I can't upload a 128 MB EXR here, so I put them side by side and took a screenshot.

<img width="2302" alt="Screenshot 2024-10-20 at 18 58 24" src="https://github.com/user-attachments/assets/b3154c4e-ad1a-43d9-abcc-af39b0d2bd46">

You might say I can't exactly judge gamma, color profiles, etc when opening in macOS Preview, but that looks the same in anything I open them into. Affinity Photo, Blender, DaVinci Resolve, doesn't matter. In fact, in the tutorial the guy says that when importing it into Blender, you don't see anything, and you have to add an exposure node, and even a gamma node. But in my case I can see it as soon as I load it with the image node, and with the exposure node I can make it less over exposed, but only to a certain extent.

But as you can see in the side by side comparison, the PNG export looks fine, just as I see it in the program when tweaking all the settings. And if I all I cared to do was to export a still, I would simply use PNG. But I saw how many beautiful things you can get by animating the different nodes, and as far as I can tell, the only option for exporting animations is EXR.

So is the way EXR exports look the way it's supposed to be?

**Comment by bezo97** [comment]
Hi! Not sure why the EXR looks off, I'll look into it soon. But to solve your problem, you don't need EXR to do animations, it works just as well with PNGs! In the Settings window, select the Animations section and make sure the export format is png as shown on this screenshot: 
![image](https://github.com/user-attachments/assets/e8d3fe10-33c3-4d44-a152-a688f5b3a7f4)

**Comment by sebarious70** [comment]
Thanks for the tip, the only problem is that the PNG exporter apparently doesn't support alpha channel, and ideally I would like to render this to a sequence that will be right above the bottom one which contains the stars, generated in Blender.

Or did I miss the setting for that too?

**Reply by MayAvali** [reply]
Hey, this isn't necessarily a solution, but it _is_ a pretty good workaround. If you use an additive blending mode in the software you are using to layer the images together, the black background will be considered transparent. Maybe this will work for you?
